src/libraries/System.Reflection/tests/TypeInfoTests.cs

```diff
@@ -471,10 +471,9 @@ public static void IsEnumDefined(object value, bool expected)
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/mono/mono/issues/15028", TestRuntimes.Mono)]
         public void IsEnumDefined_Invalid()
         {
-            AssertExtensions.Throws<ArgumentException>("", () => typeof(NonGenericClassWithNoInterfaces).GetTypeInfo().IsEnumDefined(10));
+            AssertExtensions.Throws<ArgumentException>("enumType", () => typeof(NonGenericClassWithNoInterfaces).GetTypeInfo().IsEnumDefined(10));
```

Comment:
The implementation has another path for string. It would be nice to add a test for:
`typeof(NonGenericClassWithNoInterfaces).GetTypeInfo().IsEnumDefined("10")`

and make sure that the exception is same for it.